### DISCUSSION TITLE
Create static "maintenance" HTML page

### DISCUSSION THREAD

**Discussion by jonscanzi** [clicepfl/clic-website#238] [Ideas]
This page could then be displayed by the proxy if/when the website server is unavailable.

This is fairly important, as the normal "bad gateway" message from the proxy is fairly ugly.

While this may seem somewhat pointless since both the proxy and the website are running on the same computer, this is for future-proofing if one day the proxy is separate (which is my end-goal)

**Comment by jonscanzi** [comment]
Not relevant since we will most likely not build a second server any time soon.

**Comment by mochiruntime** [comment]
Maybe we should close this for now @s314cy ? Or we could also implement something similar if we dockerize the website #134, which is displayed automatically whenever the container is down; of course this would not work if the physical server crashes.
